compositor_common/src/error.rs

```diff
@@ -100,3 +104,13 @@ impl<'a> Iterator for ErrorStack<'a> {
         })
     }
 }
+
+impl UnsatisfiedConstraintsError {
```

Comment:
ErrorStack is not a part of this error hierarchy, so I would prefer to keep it either at the very begining or end of the file